README.md

```diff
@@ -65,6 +66,7 @@
 - [The Public Lock Contract](https://thirdweb.com/unlock-protocol.eth/PublicLock) - The Public Lock Contract is a membership contract from Unlock Protocol. This contracts lets you create recurring subscriptions, tickets, certifications.
 - [Rental NFTs](https://thirdweb.com/doubledev.eth/ERC4907) - ERC-4907 facilitates the efficient implementation of rentable NFTs.
 - [ERC-721 Community Stream](https://thirdweb.com/flairsdk.eth/ERC721CommunityStream) - Allows you to create a token stream to equally distribute any tokens (Native, ERC20, etc) to holders of a specific ERC721 NFT collection.
+- [Job Application](https://thirdweb.com/goerli/0xAa3C537CDeeFc8Ea9e86b7C5dAecF11fa4cDA01E) - job application smart contract allow users to list ajob, apply for a job.
```

Comment:
Hey @joeephwild maybe use a release page version of your contract to be able to use by the community.

Reply:
okay sure will do that and rerun the pull request

Reply:
im trying to update the code for better usage

Reply:
will wait for your PR @joeephwild

Reply:
done